be/src/util/file_utils.cpp

```diff
@@ -240,16 +240,15 @@ Status FileUtils::copy_file(const std::string& src_path, const std::string& dest
     }
 
     const int64_t BUF_SIZE = 8192;
-    char* buf = new char[BUF_SIZE];
-    DeferOp free_buf(std::bind<void>(std::default_delete<char[]>(), buf));
+    std::unique_ptr<char[]> buf(new char[BUF_SIZE]);
```

Comment:
std::unique_ptr<char[]> buf = `make_unique<char[]>(BUF_SIZE);`  may be better

Reply:
fixed